hail_search/queries/base.py

```diff
@@ -192,14 +192,18 @@ def _enum_field(value, enum, ht_globals=None, annotate_value=None, format_value=
 
         return value
 
-    def __init__(self, sample_data, genome_version, sort=XPOS, sort_metadata=None, num_results=100, inheritance_mode=None, **kwargs):
+    def __init__(self, sample_data, genome_version, sort=XPOS, sort_metadata=None, num_results=100, inheritance_mode=None,
+                 override_comp_het_alt=False, **kwargs):
         self._genome_version = genome_version
         self._sort = sort
         self._sort_metadata = sort_metadata
         self._num_results = num_results
+        self._override_comp_het_alt = override_comp_het_alt
         self._ht = None
         self._comp_het_ht = None
+        self.unfiltered_comp_het_ht = None
```

Comment:
It stands out that this isn't denoted as "private".  Is there a reason?

Reply:
It's accessed directly by the multi data type query class, not as like a `super` call but by actually reading the property from an instantiated class, so I thought it was better to be explicit about the fact that it is intended to be used outside the class - https://github.com/broadinstitute/seqr/pull/3603/files#diff-e646625d305f6373c7ae24a18d8b0857c2bb1eceaa924076023103695d9f67b2R45

Reply:
👍 silly, but I'd maybe move the var declaration to be above or below all the privates.